cypress/e2e/CyberSecurity/SQL_injection.js

```diff
@@ -0,0 +1,21 @@
+/// <reference types="cypress" />
+
+const { homePage, searchQueries, url, weatherPage } = require("../../support/selectors");
+
+describe('Make sure BBC.com loads properlly', () => {
+
+    it('Test Case 38. search bar SQL injection', () => {
+        cy.get(homePage.searchIcon).click({force: true});
+        cy.get(homePage.searchBar).type(searchQueries.SQLInjectionSearchBar, { delay: 200 });
+        cy.get(homePage.searchEnterButton).click();
+        cy.get(homePage.allArticlesTitles).should('have.length.gt', 0);
+    });
+
+    it('Test Case 39. Weather search bar SQL injection', () => {
+        cy.visit(url.weatherPage);
+        cy.get(weatherPage.citySearchBar).type(searchQueries.SQLInjectionCitySearch);
+        cy.get(weatherPage.searchButton).click();
+        cy.get(weatherPage.citySearchBar).clear().type(searchQueries.vilnius);
+        cy.get(weatherPage.searchResults).should('have.length.gt', 0);
```

Comment:
bet ar cia tikrai SQL injectionai, jeigu tu tiesiog inpute parasai zodzius, ir pagal tai bando surasti relevant articles?

Reply:
jei inputas neapsaugotas tai naudotojas gali irasyti papildomos logikos kodui. pvz.: mano irasomoj eilutej: a - tai nieko is esmes nereiskiantis inputas, ' - uzdarau kabutes kurios apibrezia kad tai stringas, ; - pabaigiu eilute, DROP TABLES cities; - tai nauja SQL komanda kuri turetu istrinti visa miestu duombaze, -- uzkomentuoju visas likusiai eilutes